### ISSUE TITLE
Empty "Star" Icon (UI)

### ISSUE THREAD

**Issue by maltfield** [eloquence/lib.reviews#323]
This is a feature request to add a set of "empty star" icons -- in addition to the number of "star icons" that the user left for the review

### Problem

If the user leaves a 4-star review, then the UI presents it in a way that readers think they left a 4/4 star review. In reality, they left a 4/5 star review.

### Solution

An "empty star" icon should be created that is a dimmed-out (reduced opacity) version of the "star" icon.

* If the user leaves a 5-star review, then 5 "star" icons should appear
* If the user leaves a 4-star review, then 4 "star" icons should appear -- followed by 1 "empty star" icon
* If the user leaves a 3-star review, then 3"star" icons should appear -- followed by 2 "empty star" icons
* If the user leaves a 2-star review, then 2 "star" icons should appear -- followed by 2 "empty star" icons
* If the user leaves a 1-star review, then 1 "star" icon should appear -- followed by 4 "empty star" icons

**Comment by eloquence** [comment]
Good point @matfield. I've been working a bit with https://ura.design on redesigning different parts of the UI. Here's what they came up with for the star design:

![image](https://github.com/eloquence/lib.reviews/assets/213636/b7f32516-d882-4cc3-8942-357b04cdb8d9)

The color would still change depending on the number of stars there are, e.g.:

![image](https://github.com/eloquence/lib.reviews/assets/213636/63592d1f-84f0-438e-909e-f8f62e474322)

What do you think of this design?

**Comment by maltfield** [comment]
I think that design is wonderful. The color change is a good plus :)
